scenarios.md

```diff
@@ -230,6 +230,37 @@ A weakness is discovered in a widely used cryptographic algorithm and a decision
 1. Key revocation, chain of trust, etc. must all work for the expected lifetime of a version of the client software while these changes are made.
 1. The actions that different parties need to perform must be clearly articulated, along with the result of not performing those actions.
 
+### Scenario #10: Verifying Different Types of Artifacts
+
+Artifacts may be referenced in different ways and verification of signatures should handle the different types of artifact references:
+
+1. Immutable `sha256` digests
+1. Mutable tags
+1. Immutable tags available in some registries
+1. Tags and digests to a manifest list for multi-platform images
```

Comment:
This should note that using digests it is possible to refer both to the multi-platform manifest list _or_ to the single-platform component (and the registry API doesn’t allow looking up the manifest list from the single-platform component), and both kinds of digest references should be usable (and should be possible to sign/verify).